opentech/apply/activity/messaging.py

```diff
@@ -526,9 +529,10 @@ def batch_notify_reviewers(self, sources, links, **kwargs):
         )
 
     def slack_id(self, user):
-        if user.slack:
-            return f'<{user.slack}>'
-        return ''
+        if not user or not user.slack:
```

Comment:
Combine negative checks like so: `if not (user or user.slack):`.  It's much easier for a reader to consider `A or B` then apply the negative to it.

Reply:
This is [De Morgan's Law](https://en.wikipedia.org/wiki/De_Morgan%27s_laws) if you want to read up the reasoning behind it!

Now I think about this more I think this could be more readable as two checks returning the same thing but make the first one `if user is not None` so it's more explicit what you're confirming.